src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1200px;
+}
+
+.nav__bar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 40px 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  padding: 10px 0;
+  height: 40px;
+}
+
+.list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  position: relative;
+  list-style: none;
+}
+
+.list__item {
+  list-style: none;
+}
+
+.list__link {
+  margin: 5px;
+  padding: 23px 5px;
+  line-height: 60px;
+  color: #000;
+  text-decoration: none;
+  transition: color 0.3s;
+}
+
+.list__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  left: 10px;
+  width: 36px;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
This styles should work correct for each link, if you add class is-active for other link - it won't be correct
Use width: 100% and try to set position using position: relative and bottom properties (without top, left)